aviary/docs/user_guide/variable_metadata.ipynb

```diff
@@ -240,6 +283,19 @@
     "Let's say that we have created our `ExtendedAircraft` and `ExtendedMetaData` from above, and that elsewhere we have a subsystem that requires information about engine cooling system mass as well as whether the aircraft has winglets. Below is the buildup of the `Aircraft` type hierarchy and the metadata for our new subsystem:"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Testing Cell\n",
+    "from aviary.utils.merge_variable_metadata import merge_meta_data\n",
+    "from aviary.utils.merge_hierarchies import merge_hierarchies, merge_two_hierarchies\n",
+    "# ********NOTE********\n",
```

Comment:
Are you asking about which of `merge_hierarchies` or `merge_two_hierarchies`? If so then it should be merge_hierarchies. `merge_two_hierarchies` was never intended to be a user-facing function, it's just a helper function for `merge_hierarchies`. It doesn't include error checking, and also would add confusion since `merge_hierarchies` has the same capability but for n number of hierarchies instead of 2.

Reply:
The docs say

> The `merge_meta_data()` function has been provided to combine all the different metadata into one. The `merge_meta_data()` function behaves quite similarly to the `merge_variable_hierarchies()`

However, `merge_variable_hierarchies` no longer exists. The most similarly named functions I could find were `merge_hierarchies` or `merge_two_hierarchies`. It sounds like `merge_variable_hierarchies` should be changed to `merge_hierarchies`, is that correct?